tools/sethost.sh

```diff
@@ -211,4 +211,9 @@ fi
 sed -i -e "/CONFIG_HOST_OTHER/d" $nuttx/.config
 
 echo "  Refreshing..."
-make olddefconfig $* || { echo "ERROR: failed to refresh"; exit 1; }
+
+if [ ! -z $* ]; then
```

Comment:
How about we check V=1 instead?

Reply:
To only keep V=1? No other option is useful?

Reply:
> To only keep V=1? No other option is useful?

other option is also useful(like -j8), but V=? has the similar semantics:
1.If the extra command line contain V=1, remove "1>/dev/null"
2.Otherwise, add "1> /dev/null".